packages/react-composites/src/composites/MeetingComposite/adapter/AzureCommunicationMeetingAdapter.ts

```diff
@@ -85,52 +86,70 @@ export class AzureCommunicationMeetingAdapter implements MeetingAdapter {
     throw new Error('Method not implemented.');
   }
   public dispose(): void {
-    throw new Error('Method not implemented.');
+    this.unsubscribeMeetingEvents();
+    this.callAdapter.dispose();
+    this.chatAdapter.dispose();
   }
   public setPage(page: MeetingCompositePage): void {
-    throw new Error('Method not implemented.');
+    switch (page) {
```

Comment:
nit: isntead of switch statement this can just be:
```
export function meetingPageToCallPage(page: MeetingCompositePage): CallCompositePage {
  return page === 'meeting' ? 'call' : page;
}
```

Reply:
updated